supervisely/project/project.py

```diff
@@ -2673,24 +2673,23 @@ def download_bin(
                     geometries_list = api.image.figure.download_geometries_batch(alpha_ids)
                     alpha_geometries.update(dict(zip(alpha_ids, geometries_list)))
                 figures.update(ds_figures)
-                if log_progress:
-                    ds_progress.iters_done_report(len(batch))
-                if progress_cb is not None:
-                    progress_cb(len(batch))
-        dataset_infos = [ds for ds in dataset_infos if ds.id not in dataset_skip_list]
-        if return_bytesio:
-            data = io.BytesIO()
-            pickle.dump(
-                (project_info, meta, dataset_infos, image_infos, figures, alpha_geometries), data
-            )
-            return data
+                if log_progress or progress_cb is not None:
+                    ds_progress(len(batch))
```

Comment:
if both log_progress and progress_cb are present, then progress_cb will never be called

Reply:
Add a higher priority for progress_cb